internal/convert.go

```diff
@@ -439,7 +438,7 @@ func (conv *Conv) AddShardIdColumn() {
 			colName := conv.buildColumnNameWithBase(t, ShardIdColumn)
 			columnId := GenerateColumnId()
 			ct.ColIds = append(ct.ColIds, columnId)
-			ct.ColDefs[columnId] = ddl.ColumnDef{Name: colName, Id: columnId, T: ddl.Type{Name: ddl.String, Len: 50}, NotNull: false}
+			ct.ColDefs[columnId] = ddl.ColumnDef{Name: colName, Id: columnId, T: ddl.Type{Name: ddl.String, Len: 50}, NotNull: false, AutoGen: ddl.AutoGenCol{Name: "None", Type: "None"}}
```

Comment:
[nit] Could we make `AutoGen: ddl.AutoGenCol{Name: "None", Type: "None"}` as a constant?
Something like `DEFAULT_AUTOGEN_COL  ddl.AutoGenCol = ddl.AutoGenCol{Name: "None", Type: "None"}`